src/Module/BaseApi.php

```diff
@@ -160,15 +158,11 @@ protected static function login()
 	 */
 	protected static function getCurrentUserID()
```

Comment:
Aren't `BaseApi::getCurrentUserID()` and `BaseApi::login()` almost identical?

Reply:
So, `login` can call `getCurrentUserID` and throw an error if it's 0, right?

Reply:
It has to coexist with the legacy client credentials.

Reply:
My plan is to have a look at all these places, once the API is stabilized.